### ISSUE TITLE
Dockerized container

### ISSUE THREAD

**Issue by husw5y** [Bgi-LUSH/LUSH-DNASeq-pipeline#2]
I want to test this using docker. What are the dependencies and install instructions? how much RAM is required?

**Comment by Bgi-LUSH** [comment]
We have released the binary version of the Lush tools, which can be run in a CentOS-based Docker environment, such as centos:centos7.6.1810. For 30x WGS data, approximately 40GB of memory is required.

**Comment by husamia** [comment]
> We have released the binary version of the Lush tools, which can be run in a CentOS-based Docker environment, such as centos:centos7.6.1810. For 30x WGS data, approximately 40GB of memory is required.

that's great. Can you publish a docker container that we can pull and run?

**Comment by Bgi-LUSH** [comment]
You can use "docker pull wonderfulv1/lush-toolkit:v1" to pull the corresponding image. The respective lush tools are located in the "/usr/local/lush/LUSH-DNASeq-pipeline/bin/" directory.

**Comment by husamia** [comment]
@Bgi-LUSH do you have a command for pulling the latest in the future?

**Comment by Bgi-LUSH** [comment]
Yes, we will continue to update, and in the future, we will label the latest version as 'latest'.

**Comment by Bgi-LUSH** [comment]
hi，we have added a Dockerfile to build the latest image. Just run :
```
git clone https://github.com/Bgi-LUSH/LUSH-DNASeq-pipeline.git
docker build -t lush-toolkit:latest  -f LUSH-DNASeq-pipeline/Dockerfile  .
```
